src/partials/query.editor.html

```diff
@@ -1,5 +1,18 @@
 <query-editor-row query-ctrl="ctrl" class="cassandra-datasource-query-row" has-text-edit-mode="false">
     <div class="gf-form-inline" ng-if="!ctrl.target.rawQuery">
+
+        <div class="gf-form">
+            <label class="gf-form-label width-12">
+                Allow filtering
+                <info-popover>Allow / disallow filtering.</info-popover>
```

Comment:
LGTM but the info MUST include a warning that "Allow Filtering" is a very bad practice and we strongly discourage using it